### ISSUE TITLE
Add switch to toggle between dark mode and light mode

### ISSUE THREAD

**Issue by chessmadridista** [chessmadridista/Geektrust-Teerex-Store#18]

**Comment by Weave-Trix** [comment]
Hi, do you need help on this?

**Comment by Weave-Trix** [comment]
@chessmadridista

**Comment by chessmadridista** [comment]
Hi, @Weave-Trix! Actually, this is a Geektrust (it is a job portal) project which I need to submit on their website. So, unfortunately people other than me cannot contribute to it :(. 

Are you looking to work in open source?

**Comment by Weave-Trix** [comment]
@chessmadridista yes, please let me know if there's anything that i can contribute in.

**Comment by chessmadridista** [comment]
@Weave-Trix Unfortunately, you cannot contribute to this repo :(.

But I have a couple of personal project ideas that I will be open sourcing. I will contact you whenever that happens (in a week or so) :).
